docs/tools/pizza-cli/codeowners.md

```diff
@@ -0,0 +1,113 @@
+---
+id: codeowners
+title: "Pizza CLI Codeowners Command"
+sidebar_label: "Codeowners"
+keywords: 
+- "codeowners" 
+- "codeowners automation" 
+- "CLI codeowners" 
+---
+
+The `codeowners` command in the Pizza CLI is designed to automatically generate and manage CODEOWNERS files for your GitHub repositories. CODEOWNERS files are used to define individuals or teams responsible for code in a repository, which is useful for code review processes and maintaining code quality.
+
+Key benefits of using the codeowners command:
+
+- Automatically determines codeownership based on contribution history
+- Simplifies the process of keeping CODEOWNERS files up-to-date
+- Ensures that the right people are notified for code reviews
+- Helps in identifying subject matter experts for different parts of the codebase
+- Facilitates better project management and knowledge distribution
+
+To generate a CODEOWNERS file, you need to [install the Pizza CLI](pizza-cli.md) and run the `pizza generate codeowners` command in your terminal. However, before running the command, it's important to set up proper attribution for committers in a `.sauced.yaml` file.
+
+## Creating a .sauced.yaml file
+
+The `.sauced.yaml` file is necessary for mapping commit email addresses to GitHub usernames. However, it's important to be selective about who you include in this file. Not every contributor should be a codeowner.
+
+<details>
+<summary>Selecting appropriate codeowners</summary>
+
+When deciding who to include in your `.sauced.yaml` file, consider the following:
+
+1. **Team members**: Include active members of your organization who are responsible for maintaining the codebase.
+
+2. **Expertise**: Prioritize individuals with deep knowledge of specific areas of the codebase.
+
+3. **GitHub team members**: Include members of the GitHub teams associated with the repository.
+
+4. **Long-term contributors**: Consider long-standing contributors who have demonstrated commitment to the project.
+
+:::tip
+Be cautious about including external contributors. Only include those who have been vetted and are trusted to review and approve changes.
+:::
```

Comment:
Let's add empty lines around the starting and ending directives to avoid auto-format by Prettier or anything similar as per [this docs](https://docusaurus.io/docs/markdown-features/admonitions) on Docusaurus. 

```suggestion
:::tip

Be cautious about including external contributors. Only include those who have been vetted and are trusted to review and approve changes.

:::
```